src/main/java/core/basesyntax/Colors.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+public enum Colors {
```

Comment:
enum class names should be singular form as per [style guide](https://google.github.io/styleguide/javaguide.html#s4.8.1-enum-classes)